scripts/testnet.sh

```diff
@@ -45,17 +45,53 @@ get_rpc() {
 # $2=rpc
 wallet_balance() {
   local address=${1}
-  local rpc=${2}
+  local rpc=${2} # @TODO: Replace RPC for chain to avoid calling only xdai
 
-  yarn run --silent ethers eval "new ethers.providers.JsonRpcProvider('${rpc}').getBalance('${address}').then(b => formatEther(b))"
+  curl https://api.hoprnet.org/api/balance/xdai/$1/native/get?text=true
 }
 
 # $1=rpc
 funding_wallet_address() {
-  local rpc=${1}
+  local rpc="${1}" # @TODO: Remove RPC
 
   # the value of FUNDING_PRIV_KEY must be prefixed with 0x
-  yarn run --silent ethers --rpc "${rpc}" --account ${FUNDING_PRIV_KEY} eval 'accounts[0].getAddress().then(a => a)'
+  curl --silent https://api.hoprnet.org/api/faucet/address/get?text=true
+}
+
+# $1 = account (hex)
+# $2 = token address (hex)
+faucet_address_token() {
+  local address="${1}"
+  local token="${2}"
+  local secret="${FAUCET_SECRET_API_KEY}"
+
+  curl --silent --request POST \
+  "https://api.hoprnet.org/api/faucet/xdai/$address/$token/default" \
+  --header 'Content-Type: application/json' \
+  --data-raw "{\"secret\": \"$secret\"}"
```

Comment:
Please create ticket to track that we should change the API to use header secrets instead of POST body values.

Reply:
Alignment of the multi-line commands is off. Also below.

Reply:
Agreed on both here.